student.rb

```diff
@@ -0,0 +1,14 @@
+require_relative('./person')
+
+class Teacher < Person
+  attr_accessor :classroom
+
+  def initialize(name, age, parent_permission, classroom)
```

Comment:
- [ ] Great job implementing the `Student` class 👍 however according to the requirements `name` and `parent_permission` should have default values of `"Unknown"` and `true`.